src/tests/utilities.test.js

```diff
@@ -16,4 +21,12 @@ describe("basic utilities", () => {
 
     expect(result).toEqual(originalArrayBuffer);
   });
+
+  it("should convert an ArrayBuffer to Hex and back. Result should be the equal", async () => {
```

Comment:
This test is also testing your test-only util function and relying on the implementation of that. 

Why don't we just totally simply this test and write:
```
const original = new Uint8Array([ 0, 1, 10, 16, 240, 255 ]);
const expected = "00010a10f0ff";
expect(arrayBuffereToHex(original.buffer)).toEqual(expected);
```

Reply:
please look here https://github.com/VivyTeam/krypt-web/pull/73#discussion_r297198204, I might be wrong, in this case please put me right!